tests/baselines/reference/docker/azure-sdk.log

```diff
@@ -155,83 +73,61 @@ os-name (imported by dist-esm/utils/user-agent.js)
 Use output.globals to specify browser global variable names corresponding to external modules
 os-name (guessing 'osName')
 created dist/index.js in ?s
-@azure/service-bus (? seconds)
-Warning: You have changed the public API signature for this project. Updating review/service-bus.api.md
-Warning: typings/src/receiver.d.ts:28:9 - (TS1086) An accessor cannot be declared in an ambient context.
-Warning: typings/src/receiver.d.ts:34:9 - (TS1086) An accessor cannot be declared in an ambient context.
-Warning: typings/src/receiver.d.ts:153:9 - (TS1086) An accessor cannot be declared in an ambient context.
-Warning: typings/src/receiver.d.ts:159:9 - (TS1086) An accessor cannot be declared in an ambient context.
-Warning: typings/src/receiver.d.ts:165:9 - (TS1086) An accessor cannot be declared in an ambient context.
-Warning: typings/src/receiver.d.ts:175:9 - (TS1086) An accessor cannot be declared in an ambient context.
-Warning: typings/src/sender.d.ts:24:9 - (TS1086) An accessor cannot be declared in an ambient context.
-Warning: typings/src/serviceBusMessage.d.ts:453:9 - (TS1086) An accessor cannot be declared in an ambient context.
-================================
-BLOCKED (15)
 ================================
-@azure/identity
-@azure/core-amqp
-@azure/core-arm
-@azure/core-lro
-@azure/event-hubs
-@azure/app-configuration
-@azure/cognitiveservices-inkrecognizer
-@azure/eventhubs-checkpointstore-blob
-@azure/keyvault-certificates
-@azure/keyvault-keys
-@azure/keyvault-secrets
-@azure/storage-blob
-@azure/storage-file-share
-@azure/storage-queue
-@azure/template
+FAILURE (2)
 ================================
-FAILURE (1)
-================================
-@azure/core-http ( ? seconds)
-npm ERR! code ELIFECYCLE
-npm ERR! errno 2
-npm ERR! @azure/core-http@X.X.X build:tsc: `tsc -p tsconfig.es.json`
-npm ERR! Exit status 2
-npm ERR!
-npm ERR! Failed at the @azure/core-http@X.X.X build:tsc script.
-npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
-npm ERR! A complete log of this run can be found in:
-npm ERR!     /root/.npm/_logs/XXXX-XX-XXXXXXXXX-debug.log
-ERROR: "build:tsc" exited with 2.
-npm ERR! code ELIFECYCLE
-npm ERR! errno 1
-npm ERR! @azure/core-http@X.X.X build:lib: `run-s build:tsc build:rollup build:minify-browser`
-npm ERR! Exit status 1
-npm ERR!
-npm ERR! Failed at the @azure/core-http@X.X.X build:lib script.
-npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
-npm ERR! A complete log of this run can be found in:
-npm ERR!     /root/.npm/_logs/XXXX-XX-XXXXXXXXX-debug.log
-ERROR: "build:lib" exited with 1.
+@azure/app-configuration ( ? seconds)
+>>> @azure/app-configuration
+tsc -p . && rollup -c 2>&1 && npm run extract-api
+samples/listRevisions.ts(43,32): error TS2322: Type 'PagedAsyncIterableIterator<ConfigurationSetting, ListRevisionsPage>' is not assignable to type 'AsyncIterable<ConfigurationSetting>'.
```

Comment:
seems like a failure from the 3.7 iterator type overhaul. might this be from an old version of TS?